benchmarks/commonMain/src/kotlinx/io/BytesInputReadBenchmark.kt

```diff
@@ -0,0 +1,67 @@
+package kotlinx.io
+
+import kotlinx.benchmark.*
+import kotlinx.io.bytes.*
+
+/**
+ * Benchmark with prebuilt ByteInput
+ */
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.Throughput)
+@OutputTimeUnit(BenchmarkTimeUnit.MICROSECONDS)
+class BytesInputReadBenchmark {
```

Comment:
If this benchmark is measuring preview overhead similarly to Okio, could you please add Okio to this benchmark?

Reply:
Also, please properly document what this benchmark is expected to measure and why it is important.
Otherwise, we would end up in a situation pretty similar to what we have now :)